meta-mender-raspberrypi-demo/conf/layer.conf

```diff
@@ -13,6 +13,4 @@ BBFILE_PATTERN_mender-raspberrypi-demo = "^${LAYERDIR}/"
 
 LAYERDEPENDS_mender-raspberrypi-demo = "mender mender-demo mender-raspberrypi"
 
-INHERIT += "rpi-demo-wifi"
```

Comment:
This is not re-added anywhere. Is it ok to just remove it?

Reply:
This is similar to the question of automatic inclusion of mender-setup.  Just adding the layer should not explicitly add anything so it's generally up to the user to add the settings to local.conf.  Not sure how to handle this.

Question: is anyone using this layer?  Can it just be deprecated?

Reply:
We use it in our Yocto Raspberry Pi image, but now our onboarding is based on the image from mender-convert, so we no longer depend on it as much. However, there have been several contributions to this layer, so I assume people are using it when they test Raspberry Pi with Yocto.

Reply:
I think we should remove it (to me this layer has always been a bit out of place). 90 % of this layer is generic actually and we could merge these parts in meta-mender-demo, to provide an easy way to configure WiFi.

The Raspberry Pi specific parts are to me a workaround to a bug at some point in meta-raspberrypi and should not be necessary. `wifi` is a default distro feature and as long as you build an image that is derived from `core-image-base` you should have everything in place for wifi to work, otherwise there is a bug in meta-raspberrypi

Reply:
Yeah, I just took a look through and this does not need to be separate from meta-mender-demo.  I'll tack that onto the end of this PR.